webxr/openxr/mod.rs

```diff
@@ -895,10 +896,8 @@ impl OpenXrDevice {
                 })?[0];
 
             let secondary_extent = Extent2Di {
-                width: (view_configuration.recommended_image_rect_width / SECONDARY_VIEW_DOWNSCALE)
-                    as i32,
-                height: (view_configuration.recommended_image_rect_height
-                    / SECONDARY_VIEW_DOWNSCALE) as i32,
+                width: view_configuration.recommended_image_rect_width as i32,
```

Comment:
why did we move the downscaling?

Reply:
Keeping the original value around seems cleaner, e.g. different layer types can use different scaling. But it's an inessential change, we can go back if we like.

Reply:
Ah, seems fine.